src/components/ColorPicker/ColorPicker.vue

```diff
@@ -251,6 +251,7 @@ export default {
 		&-color-circle {
 			display: flex;
 			align-content: center;
+			align-items: center;
```

Comment:
align-content doesn't have any effect here since it is a single line flex layout with just one element, so it would be good to remove that right away imo :+1:

Reply:
> align-content doesn't have any effect here since it is a single line flex layout with just one element, so it would be good to remove that right away imo +1

https://github.com/nextcloud/nextcloud-vue/pull/1202/commits/1c51bab73d40e285e47e146d0d1eed591d578af8 Resolved :+1: